.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,40 @@
+name: Bug Report
+description: Something not behaving the way it should? Let us know!
+title: "Bug: <Title>"
+labels: ["Type: Bug"]
+body:
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: A clear description of what the bug is. Feel free to add pictures or videos (They really help!)
+    placeholder: |
+      When I do X then Y happens.
+    render: markdown
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: To Reproduce
+    description: Steps to reproduce the behavior&#58;
```

Comment:
? Should this be `behaviors`?

Reply:
YAML doesn't like using raw colons, so I had to use the HTML code for it
It's really `Steps to reproduce the behavior:`